src/assign10v1i1/ClockUI.java

```diff
@@ -1,3 +1,4 @@
+package assign10v1i1;
```

Comment:
I commented on this before already, but we really should use the folders. We'll use git tags for each iteration of our assignment. `src` should only contain our assignment, nothing else.